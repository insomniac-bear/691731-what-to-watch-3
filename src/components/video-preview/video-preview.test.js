import React from 'react';
import renderer from 'react-test-renderer';
import VideoPreview from './video-preview.jsx';

const mockFilmData = {
  filmName: `Some name`,
  posterImage: `Some url`,
  previewImage: `Some url`,
  backgroundImage: `Some url`,
  backgroundColor: `#000000`,
  filmSrc: `Some url`,
  filmPreview: `Some url`,
  genre: `Some genre`,
  release: 1999,
  runtime: 99,
  rating: 99,
  scoresCount: 99,
  describe: `Some description`,
  director: `Director`,
  actors: [`Actor`, `Actor`],
  isFavorite: false,
  isPlaying: true,
};

it(`Render VideoPreview`, () => {
  const tree = renderer
    .create(<VideoPreview
      isPlaying={mockFilmData.isPlayinf}
      filmPreview={mockFilmData.filmPreview}
      posterUrl={mockFilmData.posterUrl}
    />,
    {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
