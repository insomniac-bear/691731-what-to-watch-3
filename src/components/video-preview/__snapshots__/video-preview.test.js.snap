// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPreview 1`] = `
<div
  className="small-movie-card__image"
/>
`;
